GBDeviceInfoExample/GBDeviceInfoExample/ViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ViewController.swift
+//  GBDeviceInfoExample
+//
+//  Created by Carles Roig on 10/04/2019.
+//
+
+import UIKit
+import GBDeviceInfo_iOS
+
+class ViewController: UIViewController {
+
+    @IBOutlet private weak var deviceView: UILabel?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let model = GBDeviceInfo.deviceInfo().modelString ?? "Model not accesible"
+        let os = String(format: "Running on %d.%d", GBDeviceInfo.deviceInfo().osVersion.major,  GBDeviceInfo.deviceInfo().osVersion.minor)
```

Comment:
Could possibly consider the `GBDeviceInfo().description` which outputs a detailed description of the device info, including the OS major and minor version.